src/App.tsx

```diff
@@ -76,18 +62,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loaded && !hasError && items.length === 0 && (
```

Comment:
Fix previous comment 
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/70009205/ee221e3d-2435-40c6-993c-4e0fbcd8cb80)